terraform/modules/data_factory/ingestion-pipeline.json

```diff
@@ -0,0 +1,251 @@
+                    "secureInput": false
+                },
+                "userProperties": [],
+                "typeProperties": {
+                    "url": "https://phdi-dev-ingestion--q5apubh.bluerock-9ed363f2.centralus.azurecontainerapps.io/fhir/geospatial/geocode/geocode_bundle",
+                    "method": "POST",
+                    "body": {
+                        "value": "{\n    \"bundle\":@{activity('standardize_phone').output},\n    \"geocode_method\": \"census\"\n}",
+                        "type": "Expression"
+                    }
+                }
+            },
+            {
+                "name": "add_patient_hash",
+                "description": "Compute a hashed identifier that can be used for record linkage.",
+                "type": "WebActivity",
+                "dependsOn": [
+                    {
+                        "activity": "geocode",
+                        "dependencyConditions": [
+                            "Succeeded"
+                        ]
+                    }
+                ],
+                "policy": {
+                    "timeout": "0.12:00:00",
+                    "retry": 0,
+                    "retryIntervalInSeconds": 30,
+                    "secureOutput": false,
+                    "secureInput": false
+                },
+                "userProperties": [],
+                "typeProperties": {
+                    "url": "https://phdi-dev-ingestion--q5apubh.bluerock-9ed363f2.centralus.azurecontainerapps.io/fhir/linkage/link/add_patient_identifier_in_bundle",
+                    "method": "POST",
+                    "body": {
+                        "value": "{\n    \"bundle\":@{activity('geocode').output},\n    \"salt_str\" : \"some-dummy-salt\"\n}    ",
+                        "type": "Expression"
+                    }
+                }
+            },
+            {
+                "name": "convert_to_fhir",
+                "description": "Convert message from Hl7 or CCD to FHIR.\n",
+                "type": "WebActivity",
+                "dependsOn": [],
+                "policy": {
+                    "timeout": "0.12:00:00",
+                    "retry": 0,
+                    "retryIntervalInSeconds": 30,
+                    "secureOutput": false,
+                    "secureInput": false
+                },
+                "userProperties": [],
+                "typeProperties": {
+                    "url": "https://phdi-dev-fhir-converter.bluerock-9ed363f2.centralus.azurecontainerapps.io/convert-to-fhir",
+                    "method": "POST",
+                    "body": {
+                        "value": "{\n    \"input_data\": \"@{pipeline().parameters.message}\",\n    \"input_type\": \"@{pipeline().parameters.message_type}\",\n    \"root_template\": \"@{pipeline().parameters.root_template}\"\n}",
+                        "type": "Expression"
+                    }
+                }
+            },
```

Comment:
This may not really be an issue, but was strange to me that this was all the way down here instead of at the top.  However, I do see that it depends on [] - so that indicates it is the starting point.

Reply:
Yeah...it looks like Azure wrote this JSON in the order the steps were added the pipeline, not the order they are actually executed it. Once we have the pipeline finalized we should be able to reorganize.